Python/thread_pthread.h

```diff
@@ -188,21 +213,32 @@ PyThread_start_new_thread(void (*func)(void *), void *arg)
     pthread_attr_setscope(&attrs, PTHREAD_SCOPE_SYSTEM);
 #endif
 
+    pythread_fn *fn = (pythread_fn *)PyMem_RawMalloc(sizeof (pythread_fn));
+
+    if (fn == NULL)
+      return PYTHREAD_INVALID_THREAD_ID;
```

Comment:
PEP 7 requires { ... }

```suggestion
    pythread_fn *fn = (pythread_fn *)PyMem_RawMalloc(sizeof (pythread_fn));
    if (fn == NULL) {
      return PYTHREAD_INVALID_THREAD_ID;
    }
```